index.html

```diff
@@ -182,6 +182,14 @@ <h3>
               The <dfn>current posture</dfn>.
             </td>
           </tr>
+          <tr>
+            <td>
+              <dfn data-dfn-for="Document">[[\OverridePosture]]</dfn>
```

Comment:
* "PostureOverride" sounds better than "OverridePosture".
* As discussed in our call, it makes more sense to associate this with a top-level traversable than a Document, as the posture needs to be the same for all navigables in a tab and you're only setting the internal slot for the top-level traversable's active document anyway.

Reply:
- "OverridePosture" => "PostureOverride" done in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/f4ec72af9c58a85392930d7d73ef006176c1194c
- Document => top-level traversable : Something like this https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/9a3c613454ea244a392247beec1ea602fbc8486f ?